### ISSUE TITLE
Issue with Neon Wallet v0.0.5

### ISSUE THREAD

**Issue by cy1230** [CityOfZion/neon-wallet#166]
Hi can someone please fix the current Neon Wallet v0.0.5.  I transferred some NEO from Bittrex to my NEON wallet public address and it does not display my correct current balance.  Thankfully on Neotracker.io it does display the right balance.  Also, I can't claim the gas.  When I try, a red banner shows up across the top and says transaction failed.

Please help.

Thank you.

**Comment by LucianoCrypto** [comment]
Same here, transferred from Bittrex and neotracker.io is showing me the right balance but nothing at all on my wallet, would really love a fix soon

**Comment by dannycryptos** [comment]
I'm having the same issue; however, before transferring any Neo from Bittrex, I updated to the newest version. It showed up within minutes in the Neon Wallet. After I closed the program and reopened it, it shows a 0 balance. When I check neotogas, it shows I do have a balance still on the website but I'm really not liking that it isnt showing up in the actual wallet.

I know the transaction was recorded on the blockchain, but this makes me regret moving my Neo from the exchange. I've uninstalled and reinstalled the wallet with no prevail.

Any help would be great!
